<ui version="4.0" >
 <class>PlotMarkersTab</class>
 <widget class="QWidget" name="PlotMarkersTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="xAxisGroup" >
     <property name="title" >
      <string>x-Axis Plot Markers</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="_drawXAxisMajorTicks" >
        <property name="text" >
         <string>Draw Major Ticks</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Major Tick Spacing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="_drawXAxisMinorTicks" >
        <property name="text" >
         <string>Draw Minor Ticks</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="_drawXAxisMajorGridLines" >
        <property name="text" >
         <string>Draw Major Grid Lines</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="_xAxisMajorTickSpacing" />
      </item>
      <item row="1" column="2" >
       <widget class="QCheckBox" name="_drawXAxisMinorGridLines" >
        <property name="text" >
         <string>Draw Minor Grid Lines</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QGroupBox" name="_xAxisMajorGridGroup" >
        <property name="title" >
         <string>Major Grid Appearance</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="_Label_3" >
           <property name="text" >
            <string>Line Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="_xAxisMajorLineStyle" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_9" >
           <property name="text" >
            <string>Line Width</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="_xAxisMajorLineWidth" >
           <property name="maximum" >
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_10" >
           <property name="text" >
            <string>Line Color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="Kst::ColorButton" name="_xAxisMajorLineColor" />
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QGroupBox" name="_xAxisMinorGridGroup" >
        <property name="title" >
         <string>Minor Grid Appearance</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="_yAxisMinorLineStyleLabel_2" >
           <property name="text" >
            <string>Line Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="_xAxisMinorLineStyle" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_11" >
           <property name="text" >
            <string>Line Width</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="_xAxisMinorLineWidth" >
           <property name="maximum" >
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_12" >
           <property name="text" >
            <string>Line Color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="Kst::ColorButton" name="_xAxisMinorLineColor" />
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="yAxisGroup" >
     <property name="title" >
      <string>y-Axis Plot Markers</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="_drawYAxisMajorTicks" >
        <property name="text" >
         <string>Draw Major Ticks</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Major Tick Spacing</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="_drawYAxisMinorTicks" >
        <property name="text" >
         <string>Draw Minor Ticks</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="_drawYAxisMajorGridLines" >
        <property name="text" >
         <string>Draw Major Grid Lines</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="_yAxisMajorTickSpacing" />
      </item>
      <item row="1" column="2" >
       <widget class="QCheckBox" name="_drawYAxisMinorGridLines" >
        <property name="text" >
         <string>Draw Minor Grid Lines</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QGroupBox" name="_yAxisMajorGridGroup" >
        <property name="title" >
         <string>Major Grid Appearance</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="_Label_2" >
           <property name="text" >
            <string>Line Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="_yAxisMajorLineStyle" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Line Width</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="_yAxisMajorLineWidth" >
           <property name="maximum" >
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Line Color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="Kst::ColorButton" name="_yAxisMajorLineColor" />
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QGroupBox" name="_yAxisMinorGridGroup" >
        <property name="title" >
         <string>Minor Grid Appearance</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="_yAxisMinorLineStyleLabel" >
           <property name="text" >
            <string>Line Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="_yAxisMinorLineStyle" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Line Width</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QDoubleSpinBox" name="_yAxisMinorLineWidth" >
           <property name="maximum" >
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Line Color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="Kst::ColorButton" name="_yAxisMinorLineColor" />
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Kst::ColorButton</class>
   <extends>QToolButton</extends>
   <header>colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
